packages/lit-html/src/lit-html.ts

```diff
@@ -179,12 +174,9 @@ const comment2EndRegex = />/g;
  * " \t\n\f\r" are HTML space characters:
  * https://infra.spec.whatwg.org/#ascii-whitespace
  *
- * "\0-\x1F\x7F-\x9F" are Unicode control characters, which includes every
- * space character except " ".
- *
  * So an attribute is:
- *  * The name: any character except a control character, space character, ('),
- *    ("), ">", "=", or "/"
+ *  * The name: any character except a whitespace character, ("), ('), ">",
+ *    "=", or "/"
```

Comment:
Since we link to the HTML spec, and then don't use one of its definitions, I think it's worth pointing out here how we diverge. Maybe:

```suggestion
 *  * The name: any character except a whitespace character, ("), ('), ">",
 *    "=", or "/". Note: this is different from the HTML spec which also excludes control characters.
```